CoreUtilities/test/unit/src/au/gov/asd/tac/constellation/utilities/gui/ScreenWindowsHelperNGTest.java

```diff
@@ -0,0 +1,92 @@
+package au.gov.asd.tac.constellation.utilities.gui;
+
+
+import java.awt.Dimension;
+import java.awt.Point;
+import java.awt.Window;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/*
+ * Copyright 2010-2024 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ *
+ * @author Andromeda-224
+ */
+public class ScreenWindowsHelperNGTest {
+    static final Logger LOGGER = Logger.getLogger(ScreenWindowsHelperNGTest.class.getName());
+        
+    /**
+     * Test getMainWindowCentrePoint method of class ScreenWindowsHelper.
+     */
+    @Test
+    public void testGetMainWindowCentrePoint() {
```

Comment:
Looks like there are potentially 3 different tests sitting in this one test method. Could these be split up into their own test methods (you wouldn't then need to use `fail()`)